lib/ansible/modules/windows/win_nssm.py

```diff
@@ -36,26 +36,40 @@
     description:
       - The application binary to run as a service
       - "Specify this whenever the service may need to be installed (state: present, started, stopped, restarted)"
-      - "Note that the application name must look like the following, if the directory includes spaces:"
-      - 'nssm install service "C:\\Program Files\\app.exe\\" "C:\\Path with spaces\\"'
-      - >
-        See commit 0b386fc1984ab74ee59b7bed14b7e8f57212c22b in the nssm.git project for more info:
-        U(https://git.nssm.cc/?p=nssm.git;a=commit;h=0b386fc1984ab74ee59b7bed14b7e8f57212c22b)
+    type: path
+  description:
+    description:
+      - The description to set for the service.
+    version_added: "2.8.0"
+  display_name:
+    description:
+      - The display name to set for the service.
+    version_added: "2.8.0"
+  working_directory:
+    version_added: "2.8.0"
+    description:
+      - The working directory to run the service executable from (defaults to the directory containing the application binary)
+    aliases: [ app_directory, chdir ]
   stdout_file:
     description:
       - Path to receive output.
+    type: path
   stderr_file:
     description:
       - Path to receive error output.
+    type: path
   app_parameters:
     description:
       - A string representing a dictionary of parameters to be passed to the application when it starts.
-      - Use either this or C(app_parameters_free_form), not both.
-  app_parameters_free_form:
+      - DEPRECATED since v2.8, please use C(arguments) instead.
+      - Use either this or C(arguments), not both.
+  arguments:
     version_added: "2.3.0"
     description:
-      - Single string of parameters to be passed to the service.
+      - Parameters to be passed to the application when it starts.
```

Comment:
Probably add a new point saying this was renamed to `arguments` in `2.8` to avoid confusing some people.

Reply:
Ok